manager/controllers/app/multicluster/multi_cluster_manager.go

```diff
@@ -0,0 +1,75 @@
+package multicluster
+
+import (
+	"fmt"
+	"github.com/ibm/the-mesh-for-data/manager/apis/app/v1alpha1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/serializer"
+)
+
+type ClusterManager interface {
+	GetClusters() ([]Cluster, error)
+	GetBlueprint(cluster string, namespace string, name string) (*v1alpha1.Blueprint, error)
+	CreateBlueprint(cluster string, blueprint *v1alpha1.Blueprint) (DeploymentDetails, error)
+	UpdateBlueprint(cluster string, blueprint *v1alpha1.Blueprint) (DeploymentDetails, error)
```

Comment:
I suggest to remove `DeploymentDetails` and only add something like it if and when needed